webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -12459,6 +12459,10 @@ const firstVisibleMsg = {
                 message,
                 contact
             ) {
+                if (message?.metadata?.KM_SUMMARY === 'true') {
+                    return;
+                }
+                
```

Comment:
@deepaktiwari03 when are doing the summarize we are adding the `category: hidden` in that message from backend?

Reply:
Yes, we are receiving that but still message was message in the message box list.

Reply:
Then we can the add the check for `category: hidden` here instead of the KM_SUMMARY;
refer the visibleMessage function

Reply:
sure. will add it

Reply:
added please check now